examples/transit_vnet_common/example.tfvars

```
@@ -59,12 +58,44 @@ vmseries = {
   "fw01" = { avzone = 2 }
 }
 
-common_vmseries_version = "9.1.3"
-common_vmseries_sku     = "bundle1"
+common_vmseries_version = "9.1.6"
+common_vmseries_sku     = "bundle2"
 storage_account_name    = "pantfstorage"
 storage_share_name      = "ibbootstrapshare"
 
 files = {
   "files/authcodes"    = "license/authcodes" # authcode is required only with common_vmseries_sku = "byol"
   "files/init-cfg.txt" = "config/init-cfg.txt"
 }
+
+spoke_resource_group_name  = "example_spoke_rg"
+spoke_virtual_network_name = "example_spoke_vnet"
+spoke_address_space        = ["10.113.0.0/16"]
+
+spoke_subnets = {
+  "private" = {
+    address_prefixes       = ["10.113.0.0/24"]
+    network_security_group = "sg-private"
+    route_table            = "route_table_spoke"
+  },
+}
+
+spoke_route_tables = {
+  "route_table_spoke" = {
+    routes = {
+      "route_4" = {
+        address_prefix         = "0.0.0.0/0"
+        next_hop_type          = "VirtualAppliance"
+        next_hop_in_ip_address = "10.112.0.100"
```

This was needed for the traffic to pass when I deployed it:

```suggestion
        next_hop_in_ip_address = "10.110.0.21"
```